services/ui-src/src/utils/autosave/helpers.ts

```diff
@@ -0,0 +1,50 @@
+import { FieldValues, UseFormReturn } from "react-hook-form";
+import { Choice, DropdownChoice, EntityShape, ReportStatus } from "types";
+
+const valueChanged = (currentValue: any, storedValue: any) => {
+  return currentValue !== storedValue;
+};
+
+export const shouldAutosave = (
+  currentValue: any,
+  storedValue: any,
```

Comment:
Nit: Would probably name these like `displayValue` or `formValue` and `databaseValue` or something so it's clear.